ui/src/app/components/object-detail/object-detail.component.ts

```diff
@@ -219,9 +220,9 @@ export class ObjectDetailComponent implements OnInit {
             fb.get('spColMaxLength')?.setValue('MAX')
           }
           else if (row.spColMaxLength !== 'MAX') {
-            if ((row.spDataType === 'STRING' || row.spDataType === 'VARCHAR') && row.spColMaxLength > ColLength.StringMaxLength) {
+            if ((row.spDataType === 'STRING' || row.spDataType === 'VARCHAR') && typeof row.spColMaxLength === "number" && row.spColMaxLength > ColLength.StringMaxLength) {
               fb.get('spColMaxLength')?.setValue('MAX')
-            } else if (row.spDataType === 'BYTES' && row.spColMaxLength > ColLength.ByteMaxLength) {
+            } else if (row.spDataType === 'BYTES' && typeof row.spColMaxLength === "number" && row.spColMaxLength > ColLength.ByteMaxLength) {
```

Comment:
Why is this comparison needed? Can't the data type of `spColMaxLength` we declared upfront?

Reply:
the datatype of spColMaxLength is number | string because 'MAX' is also a valid value for max length